### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by piyush1594**
Update readme as it was not getting rendered properly.

### CODE REVIEW

**Comment by piyush1594** [comment]
I think we are good to merge this.

**Comment by oscerd** [comment]
We have this tidelift check failing.

**Comment by piyush1594** [comment]
I don't know how and where its failing, job link gives 404

File: README.md [hunk 1]
```diff
@@ -311,6 +310,16 @@ Then you can use the server like:
...
 
+|                           | OpenShift  3.6.0 | OpenShift  3.7.0  | OpenShift  3.9.0  |
+|---------------------------|------------------|-------------------|-------------------|
+| kubernetes-client 1.3.92  | +                | -                 | -                 |
+| kubernetes-client 3.0.3   | -                | ✓                 | -                 |
+| kubernetes-client 3.0.10  | ✓                | ✓                 | ✓                 |
+| kubernetes-client 3.0.11  | ✓                | ✓                 | ✓                 |
+| kubernetes-client 3.1.12  | ✓                | ✓                 | ✓                 |
+| kubernetes-client 3.2.0   | ✓                | ✓                 | ✓                 |
+| Kubernetes-client 4.0.0   | ✓                | ✓                 | ✓                 |
```
Comment (rhuss) [inline README.md]: would use a lowercase `k` here

**Comment by piyush1594** [comment]
@oscerd Tidelift not appearing now, just squash and merge it after CI passes. :tada:

**Comment by piyush1594** [comment]
merging this

**Pull request merged**